lib/evm/operation/block_information.ex

```diff
@@ -0,0 +1,94 @@
+defmodule EVM.Operation.BlockInformation do
```

Comment:
This is my nemesis. It requires the EVM to know so much about the Blocktree

Reply:
agreed :/